README.md

```diff
@@ -58,3 +58,261 @@ With J2V all the structures are discovered automatically and two files are gener
 1. Make your change
 1. Make a pull request
 1. Happy contribution!
+
+## EXAMPLE
+
+### Input: 
+```json
+{
+  "apiVersion": "v3.4",
+  "data Provider": "Eat me",
+  "restaurants": [
+    {
+      "name": "Super Burger",
+      "city": "Sydney",
+      "country": "Australia",
+      "address": "Big Street 3",
+      "currency": "AUD",
+      "menu": [
+        {
+          "dishName": "BurgerPlus",
+          "price": 10,
+          "indegrients": [
+            "Meat",
+            "Cheese",
+            "Bun"
+          ]
+        }
+      ]
+    }
+  ],
+  "headquater": {
+    "employees": 36,
+    "city": "Olsztyn",
+    "country": "Poland",
+    "building": {
+      "address": "3 Maja 10",
+      "floors": [
+        1,
+        2,
+        7
+      ]
+    }
+  },
+  "dataGenerationTimestamp": "2019-03-30T11:30:00.812Z",
+  "payloadPrimaryKeyValue": "3ab21b54-22d6-473c-b055-4430f8927d4c"
+}
+```
+
+### Ouput:
+
+#### SQL output:
```

Comment:
Do we need to document that the currently supported SQL is compatible with Snowflake, and other dialects are not (yet) supported?

Reply:
```
#### SQL output (now only Snowflake dialect supported):
```